.github/workflows/ci.yml

```diff
@@ -10,10 +10,12 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
Required for [Codecov OIDC auth](https://github.com/codecov/codecov-action#using-oidc), see below.